### ISSUE TITLE
Supress unused-argument for PasswordPolicyValidator

### ISSUE THREAD

**Issue by JonnoFTW** [PyCQA/pylint-django#357]
Following this example: https://docs.djangoproject.com/en/4.0/topics/auth/passwords/#writing-your-own-validator 

eg: 
```py
class MinimumLengthValidator:
    def validate(self, password, user=None):
        ...
```
results in an `unused-argument` error being raised by pylint for `user`. Would it be possible to have the plugin ignore this?

Versions:

```
pylint==2.12.2
pylint-django==2.5.2
pylint-plugin-utils==0.7
django-filter==2.4.0
django-oauth-toolkit==1.0.0
django-otp==1.0.6
django-render-block==0.8.1
django-templated-email==3.0.0
djangorestframework==3.13.1
pytest-django==4.5.2
```
